docs/development/features.md

```diff
@@ -66,7 +66,11 @@ Note:
 
 :heavy_check_mark: Build, Push, Pull & Scan images
 
-:calendar: Repositories management
+:calendar: Registry Configuration
+
+:sun_with_face: Repository Access Control
+
+:sun_with_face: Registry Access Control
```

Comment:
Aren't these kind of the same thing? Allowing to filter on (parts of) the repository path just allows more fine-grained control. But essentially it is just pattern matching over the full image name (including registry, repository, and tag).

Reply:
Leaving aside the implementation feasibility, 

- by "Repository Access Control" I mean the ability to allow/restrict access to, say a specific organization (namespace), or restrict pulling public images etc..
- by "Registry Access Control", I mean the ability to allow/restrict only specific registries.. Let's say I want to pull/push from/to only ghcr.io and not any other registry..

Reply:
> Leaving aside the implementation feasibility,

First off: the only feature that is somewhat meaningful is to forbid everything that isn't explicitly allowed. Denying access to specific registries/repositories (but allowing everything else) seems like a futile exercise.

> by "Repository Access Control" I mean the ability to allow/restrict access to, say a specific organization (namespace), or restrict pulling public images etc..

Allowing accesses to a repository only makes sense in the context of a specific registry. Otherwise you again allow uncontrolled access.

> by "Registry Access Control", I mean the ability to allow/restrict only specific registries.. Let's say I want to pull/push from/to only ghcr.io and not any other registry..

This makes sense. And then allowing access to only specific repositories within this registry is just a further refinement, not a separate feature.

Reply:
Ok, I will remove "Repository Access Control" treating this as just a further refinement within "Registry Access Control".